docs/tasks/tools/install-kubectl.md

```diff
@@ -141,11 +141,15 @@ kubectl cluster-info
 ```
 If you see a URL response, kubectl is correctly configured to access your cluster.
 
-If you see a message similar to the following, kubectl is not correctly configured:
+If you see a message similar to the following, kubectl is not correctly configured or not able to connect to a Kubernetes cluster locally or remotely.
```

Comment:
`locally or remotely` is a bit redundant - I think this might be better, and still highly accurate, to end it with "not able to connect to a Kubernetes cluster"